pkg/render/render.go

```diff
@@ -24,7 +22,6 @@ type Data struct {
 }
 
 type Config struct {
-	*mcfgv1.ControllerConfigSpec
```

Comment:
I don't get why this was removed

Reply:
That's a miss on my part, has to do with some experiments I've tried and forgot to revert.
good catch.

Reply:
However looking at the change I'm not sure it's wrong in that context.
It eliminates the dependency in `machine-config-operator` and it looks like this config isn't used.